### ISSUE TITLE
update for WCAG 2.2 conformance once finalized

### ISSUE THREAD

**Issue by wittjeff** [edx-solutions/xblock-drag-and-drop-v2#300]
https://www.w3.org/WAI/WCAG22/Understanding/dragging-movements
WCAG 2.2 is expected to be finalized ~December 2022.

A new accessibility requirement is designed to make it possible to drag without actually dragging; one should be able to perform the action with tap/touch/click events. I suggest the solution to this is to show four buttons (up, down, left, right arrow icons; or just up, down icons) surrounding an item's bounding box, when it is in Grabbed state.  The user can then click on the buttons to move the item. 

Given that we want this xBlock to work for people who have vision impairments, I think it might be best to have the arrow button actions move the grabbed item sequentially through the potential drop targets (centered on each drop zone).  For that to work it might make most sense to have only two arrow buttons (to avoid missing any drop targets).  Thoughts?

With this enhancement (or whatever is done to satisfy this WCAG requirement), I suggest bumping the version number to 3.

**Comment by Agrendalath** [comment]
@wittjeff, you've mentioned [on the forum](https://forum.opencraft.com/t/core-contributor-for-the-drag-and-drop-xblock/1464) that you would like to meet to discuss these updates. Do we have some more resources regarding this functionality? If yes, would you mind linking them to this GitHub issue?
